pallets/runtime/develop/src/runtime.rs

```diff
@@ -199,6 +200,21 @@ impl pallet_babe::Trait for Runtime {
     type EpochDuration = EpochDuration;
     type ExpectedBlockTime = ExpectedBlockTime;
     type EpochChangeTrigger = pallet_babe::ExternalTrigger;
+
+    type KeyOwnerProofSystem = Historical;
```

Comment:
Could you please add the doc comments here to justify the use of these types as these are not too much self-explanatory

Reply:
and same for below ones